### DISCUSSION TITLE
How to use Remove Options _ro in a multi-choice field based of KtlCond

### DISCUSSION THREAD

**Discussion by SteveRobinson-FIShR** [cortexrd/Knack-Toolkit-Library#353] [Q&A]
Hi All

I am relatively new to KTL and still trying to figure out how to insert keywords with KTL conditions.  In this example I have a multi-choice field with four options and I want to remove one of the choices for everyone without a manager role.  Can this be done?  

Thanks in advance for any help.

Cheers

Steve

**Comment by CSWinnall** [comment]
Hi @SteveRobinson-FIShR 

Welcome to the KTL.
Yes this can be done this should work:

```
_ro=[Field Title, Option 1, Option 2, Option 3, etc], [ktlRoles, !Manager]
```

Craig

**Reply by SteveRobinson-FIShR** [reply]
Thanks Craig

Given I want to remove only one choice from everyone other than the manager role do I include just the option that is only available to managers in the key word phrase or should the key word phrase be written with the option to be removed and the roles they cannot access (see below).

Eg 

_ro=[Status, closed], [ktlRoles, !Manager] - Manager only can see Closed option

or

_ro=[[Status, closed], [ktlRoles, !Employee, !Client, !Admin]  - Employee, Client and Admin cannot view Closed option

**Reply by CSWinnall** [reply]
The first one will remove it for everyone but Manager

The second one will remove it from everyone except employee client admin

If you want to remove it from employee or client or admin

You would remove the ! from before each user role.
